lib/avromatic.rb

```diff
@@ -27,6 +28,10 @@ def self.configure
     eager_load_models!
   end
 
+  def self.use_encoding_providers?
+    use_custom_datum_writer && !defined?(Avromatic::Patches::SchemaValidatorPatch).nil?
```

Comment:
Can `!defined?(Avromatic::Patches::SchemaValidatorPatch).nil?` be simplified to `defined?(Avromatic::Patches::SchemaValidatorPatch)?`?